src/main/java/com/ldtteam/structurize/placement/StructurePlacer.java

```diff
@@ -439,6 +443,11 @@ public BlockPlacementResult handleEntitySpawn(
                                 continue;
                             }
 
+                            if (entity instanceof Display.TextDisplay && (!handler.isCreative() || handler.fancyPlacement()))
```

Comment:
agree, but why only text display?

Reply:
Just thinking about it. Don't we have a similar check somewhere about LivingEntities already? Could add TextDisplay to the list there

Reply:
since possibly you could do visual stuff with block/item display didnt want to block those inherently

Reply:
> Just thinking about it. Don't we have a similar check somewhere about LivingEntities already? Could add TextDisplay to the list there
havent seen anything like that :D